solution/local.defaults.properties

```diff
@@ -0,0 +1 @@
+GOOGLE_MAPS_API_KEY=your_maps_api_key
```

Comment:
We are using `DEFAULT_API_KEY` for the stand in
GOOGLE_MAPS_API_KEY=DEFAULT_API_KEY